.github/workflows/codebuild-ci.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v1
         with:
           role-to-assume: ${{ secrets.CI_AWS_ROLE_ARN }}
+          role-duration-seconds: 120
```

Comment:
Two minutes is adequate? This doesn't have to match the hour that it takes the job to run?

Reply:
Fixed